processing/src/test/java/io/druid/query/groupby/GroupByQueryRunnerTest.java

```diff
@@ -1152,6 +1191,62 @@ public void testGroupByWithOrderLimit4()
   }
 
   @Test
+  public void testPostAggHavingSpec()
```

Comment:
this only tests post-aggregators in the context of merging results.
Can we add a separate one that does it without merging, i.e. that checks the results of GroupByQueryRunnerTestHelper.runQuery directly?